libs/ui/src/lib/symbol-autocomplete/symbol-autocomplete.component.html

```diff
@@ -12,6 +12,11 @@
   (optionSelected)="onUpdateSymbol($event)"
 >
   @if (!isLoading) {
+    @if (!empty && !filteredLookupItems.length) {
```

Comment:
Technically this works perfectly. However, I would not show the message for a single character. As the search only provides results for a [query length > 1](https://github.com/ghostfolio/ghostfolio/blob/main/libs/ui/src/lib/symbol-autocomplete/symbol-autocomplete.component.ts#L123), it is a bit annoying.

Reply:
You are right.
So for fixing this, i introduced a new variable. The reason behind that is, that checking for the input being larger than 1 character results in the message popping up for the debounce time of 400ms, that looked wrong to me. So now a flag is set after the loading is done, inside the subscribe, which then checks for the result length. That way it works quite nicely.